dashboard/test/controllers/script_levels_controller_test.rb

```diff
@@ -26,6 +26,8 @@ class ScriptLevelsControllerTest < ActionController::TestCase
     @custom_s2_l2 = create(:script_level, script: @custom_script,
                            stage: @custom_stage_2, :position => 2)
     client_state.reset
+
+    Gatekeeper.clear
```

Comment:
I don't know what the Gatekeeper is, or what this line does.  Will you explain this?